vmdb/lib/miq_automation_engine/service_models/miq_ae_service_orchestration_stack.rb

```diff
@@ -28,5 +28,13 @@ def remove_from_vmdb
       @object = nil
       true
     end
+
+    def raw_exists?
+      status, _reason = raw_status
+      status.nil? || status.downcase == 'delete_complete' ? false : true
+    rescue => err
+      return false if err.to_s =~ /[S|s]tack.+does not exist/
+      raise
+    end
```

Comment:
@bzwei This logic should be in the active_record model and then just have an `expose` for the method here.